### PR TITLE
Secvul

### PR DESCRIPTION
**Pull request by jburel**
In this PR
* Update the link so we go directly to the new website
* Update term to match web site

 Ref in history updated in https://github.com/openmicroscopy/ome-documentation/pull/1769

### CODE REVIEW

File: common/conf.py [hunk 1]
```diff
@@ -169,8 +169,7 @@ def get_previous_version(majornumber, minornumber=0):
     'djangodoc' : ('https://docs.djangoproject.com/en/1.8/%s', ''),
     'doi' : ('http://dx.doi.org/%s', ''),
     'pypi': ('https://pypi.python.org/pypi/%s', ''),
-    # Remaining Plone links
-    'secvuln' : (oo_root + '/info/vulnerabilities/%s', '')
+    'secvuln' : (oo_root + '/security/advisories/%s', '')
```
Comment (sbesson) [inline common/conf.py]: Worth moving up alonsgide with the rest of the website extlink?

File: omero/sysadmins/server-security.rst [hunk 2]
```diff
@@ -21,7 +21,7 @@ client including:
 .. note:: The OMERO team treats the security of all components with care and
     attention. If you have a security issue to report, please do not hesitate
     to contact us using our private, secure mailing list as described on the
-    :secvuln:`security vulnerabilities <>` page.
+    :secvuln:`Security Advisories <>` page.
```
Comment (sbesson) [inline omero/sysadmins/server-security.rst]: This should actually link to the top-level  http://www.openmicroscopy.org/security/ page.

**Review by sbesson — COMMENTED** [review]
The link in the `server-security` page should be reviewed. Additionally, as we are moving away from the security vulnerabilities semantics in favor of the security advisories, I wonder if we should consider deprecating/aliasing the `secvuln` extlink in favor of a more general `security` one.

File: omero/sysadmins/server-security.rst (hunk 2 above)
Comment (jburel) [reply omero/sysadmins/server-security.rst]: In that case we will need a new link, we cannot use ``:secvuln:``

**Comment by hflynn** [comment]
Happy with this now @sbesson ?
We can always move towards using the new `:sec:` with `advisories/xxxx` in future and deprecate the secvuln ext link after this release cycle.

**Comment by sbesson** [comment]
I think the latest state of this PR is fine and indeed the new security alias allows us to deprecate `secvuln` in the future if we want to. Only minor interrogation was whether we should name it `security` rather than `sec` but I don't think this should block this PR unless someone else has a strong opinion.

**Comment by jburel** [comment]
I will rename it now

**Pull request merged**